src/mudlet.pro

```diff
@@ -111,19 +111,23 @@ VERSION = 4.17.2
 
 # if you are distributing modified code, it would be useful if you
 # put something distinguishing into the MUDLET_VERSION_BUILD environment
-# variable to make identification of the used version simple
-# the qmake BUILD variable is NOT built-in
-BUILD_TEST = $$(MUDLET_VERSION_BUILD)
-isEmpty( BUILD_TEST ) {
+# variable (it should use '-' as the first character) to make identification of
```

Comment:
Personally I use `-local` for my personal builds but there is no quick and easy way to ensure that this separator is present in a cross-platform way that I can see.